sql_schema/BookMyShow.sql

```diff
@@ -0,0 +1,50 @@
+CREATE DATABASE Movies;
+
+USE Movies;
+
+CREATE TABLE Customers(
+CustomerID INT PRIMARY KEY,
+CustomerName VARCHAR(70) NOT NULL,
+Phone CHAR(10) NOT NULL,
+Email VARCHAR(50) NOT NULL);
+
+CREATE TABLE Theater(
+TheaterID INT PRIMARY KEY,
+TheaterName VARCHAR(30),
+TheaterLocation VARCHAR(100),
+TheaterDescription VARCHAR(200));
+
+CREATE TABLE Movies(
+MovieID INT PRIMARY KEY,
+MovieName VARCHAR(60) NOT NULL,
+TheaterID INT NOT NULL,
+FOREIGN KEY (TheaterID) REFERENCES Theater(TheaterID));
+
+CREATE TABLE DateAndTime(
+ShowID INT PRIMARY KEY,
```

Comment:
Releavance of this table ?

Reply:
A Movie can have mutiple date and time for showtime. So, i thought creating new table for storing date and time helps the same.

Reply:
what does this indicate in real life